docs/pbc/all/search/202212.0/third-party-integrations/configure-algolia.md

```diff
@@ -0,0 +1,116 @@
+The average export speed is around *100 products per minute*.
+
+{% endinfo_block %}
+
+<figure class="video_container">
+    <video width="100%" height="auto" controls>
+    <source src="https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-integration.mp4" type="video/mp4">
+  </video>
+</figure>
+
+
+{% info_block warningBox "Verification" %}
+
+Verify that your index is populated with data from your store:
+1. Go back to the Algolia website. 
+2. In the side pane, go to **Search&nbsp;<span aria-label="and then">></span> Index**. 
+3. Make sure that the index is now populated with data from your store.
+
+{% endinfo_block %}
+
+![algolia-index-data](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/algolia/integrate-algolia/algolia-index-data.png)
+
+For details about the created index data, see [Indexes](/docs/pbc/all/search/{{page.version}}/third-party-integrations/algolia.html#indexes).
+
+## Optional: Adjust Algolia configuration
+
+The default Algolia app configuration mimics the default Spryker search configuration. However, you may want to adjust some of those settings to your needs.
+
+### Overview of searchable attributes
+
+![algolia-searchable-attributes](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/third-party-integrations/configure-algolia/algolia-searchable-attributes.png)
+
+Algolia's **Searchable attributes** configuration determines which attributes are used to find results while searching with a search query.
+
+Default fields for searchable attributes are the following:
+- `sku`
+- `name`
+- `description`
+- `keywords`
+
+### Adjust the searchable attributes list
+
+1. In the side pane, go to **Search&nbsp;<span aria-label="and then">></span> Index**. 
+2. Open the Algolia indices list and find all primary indices. 
+3. On the **Configuration** tab, select **Searchable attributes**. 
+4. To adjust the **Searchable attributes** list, add and remove needed searchable attributes.
+5. Click **Review and save settings**. This opens the **Review and save settings** window.
+6. Enable **Copy these settings to other indices and/or replicas** and click **Save settings**.
+
+### Overview of facets list
+
+![algolia-facets](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/search/third-party-integrations/configure-algolia/algolia-facets.png)
+
+Algolia **Facets** configuration determines which attributes are used for search faceting.
+
+Default attributes for faceting are as follows:
+- `attributes.brand`
+- `attributes.color`
+- `category`
+- `label`
+- `prices`
+- `rating`
+
+The `prices` attribute is an object with nested fields so Algolia is creating facets for each nested field value effectively settings up faceting for all currencies and pricing modes present in product entities.
+
+### Add new attributes for faceting
+
+1. In the side pane, go to **Search&nbsp;<span aria-label="and then">></span> Index**. 
+2. Find all primary indices.
```

Comment:
I don't get this step. What do you mean by "primary indices"?

Reply:
This step is done in Algolia, the behavior in Algolia is that you have a primary index and many replicas. Those replicas have different sorting logic only. This reduces the amount of copy-paste needed in Algolia to update the same configuration.